idaes/core/solvers/tests/test_solvers.py

```diff
@@ -75,6 +75,9 @@ def test_clp_available():
 
 
 @pytest.mark.unit
+@pytest.mark.skipif(
```

Comment:
Just to check, do we have separate checks for if these solvers are available? It would be good to have one test that fails for each of these to let users know they don't have a given solver.

Reply:
Yeah, that's right.  There are two checks 1) that it's available and 2) that it solves a small test problem. The first will fail if the solver is not available.